src/view.tsx

```diff
@@ -32,7 +32,7 @@ export class HabiticaSyncView extends ItemView {
         )
     }
 
-    async onClose(){
-        ReactDOM.unmountComponentAtNode(this.containerEl.children[1]);
-    }
+    // async onClose(){
```

Comment:
This is basic template as to maintain views, can I ask why you've commented them?

Reply:
thanks for you reply.
- habitica  has the panel called "Rewards",so i add this panel in this plugins.hope this plugins to fully function.but may be have some humanized way to closed it as wish.
- the console.log is a mistake when i pull the request. by the way, i hope the plugin support i18n.
- "async onClose()" i get the error from console, so i annotated it.

Reply:
sorry about that.
I found i did not upload the reward panel. reject this pull requests.
I will be back.